.github/workflows/open-tickets.yml

```diff
@@ -77,8 +77,49 @@ jobs:
         run: vendor/bin/phpstan analyse --no-progress --level=2 --configuration=phpstan.neon
         working-directory: centreon-open-tickets
 
+      - name: Run of phpstan on /src at level max
+        run: vendor/bin/phpstan analyse --no-progress --level=9 --configuration=phpstan.core.neon
+        working-directory: centreon-open-tickets
+
+  backend-unit-test:
+    runs-on: ubuntu-22.04
+    needs: [get-version]
+    if: ${{ needs.get-version.outputs.stability != 'stable' }}
+
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@c665c7a15b5295c2488ac8a87af9cb806cd72198 # v2.30.4
+        with:
+          php-version: 8.1
+          coverage: none
+        env:
+          runner: ubuntu-22.04
+
+      - name: Install dependencies
```

Comment:
we may want to avoid using too many different ways of setup actions and instead use this to avoid too much maintenance on CI https://github.com/centreon/centreon/blob/82b8783014ae109e299569761c046bdcc1f5891c/.github/actions/backend-dependencies/action.yml#L21